magento/magento2
# Fix: The Credit Memo Items' base_weee_tax_applied_row_amnt Value is Incorrect

The setBaseWeeeTaxAppliedRowAmnt() function should be used instead of setBaseWeeeTaxAppliedRowAmount, as the database table field name is base_weee_tax_applied_row_amnt. Due to this error, the value of the sales_creditmemo_item.base_weee_tax_applied_row_amnt database column is always set to the order item value, which is incorrect.


### Manual testing scenarios (*)
1. Backend: Create a product with a $50 Fixed Product Tax (FPT).
2. Frontend: add 5 quantities of this product to your cart.
3. Frontend: Make sure that the FPT are applied and place the order.
4. Backend: Create a credit memo for only a single quantity.

**Expected Result:**
Since only a single quantity was credited, the values saved in both the sales_creditmemo_item.base_weee_tax_applied_amount and sales_creditmemo_item.base_weee_tax_applied_row_amnt columns should be 50.

**Actual Result:**
The values saved in the sales_creditmemo_item.base_weee_tax_applied_amount is 50 and sales_creditmemo_item.base_weee_tax_applied_row_amnt columns is 250 (5x 50).

### Questions or comments
A developer must look at the base_weee_tax_applied_row_amnt column of the sales_creditmemo_item database table in order to confirm the result.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#38765: Fix: The Credit Memo Items' base_weee_tax_applied_row_amnt Value is Incorrect
